jerry-main/main-unix.c

```diff
@@ -144,7 +144,7 @@ print_unhandled_exception (jerry_value_t error_value)
   }
   err_str_buf[err_str_size] = 0;
 
-  jerry_port_log (JERRY_LOG_LEVEL_ERROR, "Script Error: unhandled exception: %s\n", err_str_buf);
+  jerry_port_log (JERRY_LOG_LEVEL_ERROR, "Unhandled Exception: %s\n", err_str_buf);
```

Comment:
My problem with this is the too many colons in the output.

`Script Error: unhandled expection: SyntaxError: ...`

IMHO, "Unhandled Exception" is obviously refers to the source of the error. What about 'Unhandled Script Error: SyntaxError: ...' or only 'Script Error: SyntaxError: ...'?